misc/testerina/modules/testerina-core/src/main/ballerina/src/test/assert.bal

```diff
@@ -19,6 +19,7 @@ import ballerina/java;
 const string assertFailureErrorCategory = "assert-failure";
 const string arraysNotEqualMessage = "Arrays are not equal";
 const string arrayLengthsMismatchMessage = " (Array lengths are not the same)";
+const int maxArgLength = 2048;
```

Comment:
This is too much to be effective shall we use 80 which is usually the default character width of a terminal. Or we can do 180 as per https://en.wikipedia.org/wiki/Characters_per_line

Reply:
Yes, true. Let's use 80 then.